data/campaigns/Eastern_Invasion/scenarios/02_The_Escape_Tunnel.cfg

```diff
@@ -1,530 +1,625 @@
-            [/note]
-#endif
-        [/objectives]
-
-        [recall]
-            id=Dacyn
-        [/recall]
-    [/event]
-
-    [event]
-        name=start
-
-        [message]
-            speaker=Gweddry
-            message= _ "Where are we? I can’t see where we are going."
-        [/message]
-
-        [message]
-            speaker=Dacyn
-            message= _ "This is an old escape route for the outpost. Unfortunately these caves are currently inhabited by trolls, who are blocking our path through the tunnel."
-        [/message]
-
-        # Lift shroud around the amulet
-        [scroll_to]
-            x,y=12,2
-        [/scroll_to]
-        [remove_shroud]
-            side=1
-            x,y=12,2
-            radius=1
-        [/remove_shroud]
-        [delay]
-            time=750
-        [/delay]
-
-        # Path leading through the troll cave
-        [scroll_to]
-            x,y=15,7
-        [/scroll_to]
-        [remove_shroud]
-            side=1
-            x,y=15,7
-            radius=1
-        [/remove_shroud]
-        [delay]
-            time=750
-        [/delay]
-
-        [scroll_to]
-            x,y=22,5
-        [/scroll_to]
-        [remove_shroud]
-            side=1
-            x,y=22,5
-            radius=1
-        [/remove_shroud]
-        [delay]
-            time=750
-        [/delay]
-
-        [scroll_to]
-            x,y=28,6
-        [/scroll_to]
-        [remove_shroud]
-            side=1
-            x,y=28,6
-            radius=1
-        [/remove_shroud]
-        [delay]
-            time=750
-        [/delay]
-
-        [scroll_to]
-            x,y=34,5
-        [/scroll_to]
-        [remove_shroud]
-            side=1
-            x,y=34,5
-            radius=1
-        [/remove_shroud]
-        [delay]
-            time=750
-        [/delay]
-
-        [message]
-            speaker=Dacyn
-            message= _ "We must make haste to escape before the undead catch up."
-        [/message]
-
-        [message]
-            speaker=Gweddry
-            message= _ "Wait, before we go anywhere — who were those undead?"
-        [/message]
-
-        [message]
-            speaker=Dacyn
-            message= _ "I will explain later. For now, suffice to say they are much too powerful for us; our only hope is to head north."
-        [/message]
-    [/event]
-
-    [event]
-        name=moveto
-        [filter]
-            x,y=29,10
+#
+# Simple and easy, especially compared to S01.
+#
+# This is beatable without any combat or recruiting - just run straight through the dwarfish tunnels, and wade through the river.
+# More ambitious players can take the north path and raid the troll trezzur hole for some gold and a powerful item
+# REALLY ambitous players can fight their way through the troll leader for extra XP.
+#
+
+#define TURNS
+    {ON_DIFFICULTY 15 13 11}#enddef # undead arrival plus 3. I don't like tight turn limits, but couldn't think of another way to discourage farming the undead
```

Comment:
farming the undead is not worth discouraging IMO; having a pleasant player experience is more important, and tight turn limits go strongly against that

Reply:
I agree with you that player experience is key - I don't think the turn limit is excessively tight (I've never had trouble with it in any of my playthroughs), but if you've played the scenario and feel it's too oppressive I'd have no problem increasing it a small amount and reducing the initial gold in the next scenario to compensate.

But I do think it's important for the player experience that the scenario's gameplay matches it's lore. This is a scenario where you're supposed to be running away as fast as you can, so I'd like the scenario's mechanics to continue reflecting that.